docker/postlaunchconfig_update.sh

```diff
@@ -170,5 +171,10 @@ openstack-config --set /etc/keystone/keystone.conf \
                  spassword sndfa_time_window $SPASSWORD_SNDFA_TIME_WINDOW
 
 # Ensure db is migrated to current keystone version
-/usr/bin/keystone-manage db_sync
-/usr/bin/keystone-manage db_sync --extension spassword
+echo "[ postlaunchconfig_update - db_sync --migrate ] "
+/usr/bin/keystone-manage db_sync --migrate
+
+
+kill -9 $keystone_all_pid
+#kill -9 $keystone_admin_pid
```

Comment:
Lefover?

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/136/commits/25a077d47c9688b78c434359e78411fb11a6ba39